portfolio/src/main/java/com/google/sps/servlets/MessageServlet.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+import com.google.gson.Gson ;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/message")
+public class MessageServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<String> messages= new ArrayList<>();
```

Comment:
Run a formatter on this file, e.g. needs space before `=`

Reply:
@zyelcrier still need a space before `=`

Reply:
can you clarify this for me i dont get what your pointing at  @ljfranklin

Reply:
by formatter you mean prettier?